### DISCUSSION TITLE
Balancing Growth and Sustainability - Avalonia's Financial Evolution

### DISCUSSION THREAD

**Discussion by MikeCodesDotNET** [AvaloniaUI/Avalonia#13926] [General]
In the interest of transparency and openness, we've created a [blog post](https://avaloniaui.net/Blog/balancing-growth-and-sustainability-avalonia-s-financial-evolution) that discusses our work to create a sustainable business from Avalonia. 

I hope you enjoy the read!

**Comment by thevortexcloud** [comment]
Out of curiosity what was the advertising money spent on? 

> and some unreleased products that you'll learn more about soon! 

Does this mean things like paid controls are going to be released? Or is it some other service/subscription product?

Otherwise this is an interesting read. Thanks for sharing.

**Reply by MikeCodesDotNET** [reply]
Paid advertising was spent on the following: 

**Google Ad:** 15.31%
**Twitter Ads:** 3.80%
**LinkedIn Ads:** 31.37%
**Reddit Ads:** 33.65%

The remaining percentage (15.87%) was spent on activities including sponsoring YouTube content.

**Comment by Sorien** [comment]
fascinating reading, it's nice to see that growth ... just one question income from opencollective is counted in that 1,9% GitHub donations?

**Reply by MikeCodesDotNET** [reply]
OpenCollective wasn't counted as it doesn't come to the company. We do appreciate those donations though, as we can often purchase equipment for contributors!

**Comment by robloo** [comment]
Great post! Lots of good details and information. Should help everyone understand some of the nuances around funding.

I think the key is really just to have solid software and wait until you hit critical mass in terms of users (captain obvious I know). But open source software always seems to have less income per user than commercial closed source software. However, open source also has the ability to grow much larger than closed source in the long term. So less from more hopefully evens things out medium term and takes off strong in the long term.

Would still pay for Avalonia Pro tools and things too!

**Comment by rabbitism** [comment]
I should ask you if you are interested in sponsoring .NET Conf China this year...

**Reply by MikeCodesDotNET** [reply]
This year? You've an event before Jan 1st?

**Comment by JaykeBird** [comment]
This was a really fascinating read! I'm glad that XPF has been already successful and looks promising for the next year. I can understand the struggle of trying to determine what to make freely available for everyone and what to hold behind a paywall; personally, I think the current solution of XPF being paid and the Avalonia UI framework being free has been a good compromise. Anyway, thank you for the transparency on this!
